### PR TITLE
KAFKA-10841 LogReadResult should be able to converted to FetchPartiti…

### PR DESCRIPTION
**Pull request by g1geordie**
I put the conversion logic into LogReadResult named toFetchPartitionData.

### CODE REVIEW

**Comment by g1geordie** [comment]
@chia7712  can you help me take a look ?

File: core/src/main/scala/kafka/server/ReplicaManager.scala [hunk 1]
```diff
@@ -111,6 +111,17 @@ case class LogReadResult(info: FetchDataInfo,
     case Some(e) => Errors.forException(e)
   }
 
+  def toFetchPartitionData( isReassignmentFetch : Boolean) : FetchPartitionData = FetchPartitionData (
```
Comment (chia7712) [inline core/src/main/scala/kafka/server/ReplicaManager.scala]: Could you remove those space?

**Review by chia7712 — COMMENTED** [review]
@g1geordie Thanks for your patch.

File: core/src/main/scala/kafka/server/ReplicaManager.scala (hunk 1 above)
Comment (g1geordie) [reply core/src/main/scala/kafka/server/ReplicaManager.scala]: Thank you

**Review by ijuma — APPROVED** [review]
LGTM, thanks.

**Pull request merged**